vegassist.js

```diff
@@ -12,7 +12,15 @@ var DELAYTIME = 1000 * 60 * 2;
 // Load filters from all files in the filters directory
 var settingsFilteredTerms = settings.FILTERED_TERMS || [];
 var settingsFilters = settings.FILTERS || [];
-var filter = new TweetFilter('filters', settingsFilteredTerms, settingsFilters);
+
+var handle = settings.HANDLE || "";
+if (handle.length > 0){
+	if (handle.indexOf('@') != 0)
+		handle = '@' + handle;
+	util.track(handle);
```

Comment:
Yeah, it's worth noting though that this will retweet everything tweeted at the bot though. As you mentioned in the issue, there _is_ some potential for misuse, but we can keep an eye on it. Using the settings file this way makes it easy to temporarily pull this functionality without having to comment out code in the event that it does get abused somehow.